web/cobrands/sass/_layout.scss

```diff
@@ -264,40 +289,39 @@ body.mappage {
     margin: 0.9em 0.25em 0.85em;
   }
 
-  .nav-wrapper{
-    .nav-wrapper-2{
-      position: fixed;
-      background: $map_nav_bg;
-    }
-  }
-
-  #site-logo{
-    position: fixed;
-  }
+  #map_box {
```

Comment:
#map_box only ever appears on a .mappage so this can be moved out to the top level, and/or the top level #map_box styling which is all overridden can be removed.